web/modules/custom/joinup_search/joinup_search.behat.inc

```diff
@@ -55,23 +55,47 @@ class SearchSubContext extends DrupalSubContextBase implements DrupalSubContextI
    * @param string $type
    *   The text of the content tab.
    *
-   * @return \Behat\Mink\Element\NodeElement
-   *   The link as node element.
-   *
    * @throws \Exception
    *   Thrown when the tab is not found in the page.
    *
    * @Given the :type content tab is displayed
    */
-  public function assertContentTabDisplayed(string $type): NodeElement {
-    $xpath = "//div[contains(concat(' ', normalize-space(@class), ' '), ' tab__container ')]//span[@class='tab__text--lower'][normalize-space(string(.)) = '$type']" .
-      "/ancestor-or-self::a[@class and contains(concat(' ', normalize-space(@class), ' '), ' tab--content-type ')]";;
+  public function assertContentTabDisplayed(string $type) {
+    if (!$this->findContentTab($type)) {
+      throw new \Exception("The tab '$type' was not found in the page but it should be displayed.");
+    }
+  }
 
-    if (!$link = $this->getSession()->getPage()->find('xpath', $xpath)) {
-      throw new \Exception("The tab $type was not found in the page.");
+  /**
+   * Checks if a specific tab facet is not displayed on the page.
+   *
+   * @param string $type
+   *   The text of the content tab.
+   *
+   * @throws \Exception
+   *   Thrown when the tab is not found in the page.
+   *
+   * @Given the :type content tab is not displayed
+   */
+  public function assertContentTabNotDisplayed(string $type) {
```

Comment:
I think this step is the same as ::assertContentTypeFacetItemsNotPresent of the same class (the other supports multiple labels) and is not needed. 
The same applies for it's usage on the steps.

Reply:
If you think we can keep it though for a single label check, I am fine with it.

Reply:
I was thinking that it's the pair of `assertContentTabDisplayed ()`. Usually we have the "exists" and "not exists" pair. I missed the other one. No strong opinion.